securedrop/requirements/develop-requirements.in

```diff
@@ -19,7 +19,7 @@ pip-tools>=1.10
 pylint
 pytest-xdist
 python-vagrant
-safety
+safety>=1.8.4
```

Comment:
Just a note, we should probably switch to installing whatever the latest `safety` is in CI, rather than pinning an old version.

Reply:
tracking in #3442, was trying to make minimal changes here to get CI to pass